pkg/ako-operator/config_envvar_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
nit: we are in 2022 now :)

Reply:
fixed. good catch!

Reply:
btw. Should we update all files' headers to 2022?

Reply:
We only need to update files we modified this year. :)